be/src/runtime/memory/mem_tracker_limiter.cpp

```diff
@@ -487,21 +487,14 @@ int64_t MemTrackerLimiter::free_top_overcommit_query(
 }
 
 int64_t MemTrackerLimiter::tg_memory_limit_gc(
```

Comment:
tg_hard_memory_limit_gc
-> memory_limit_gc
-> tg_memory_limit_gc
But free_top_memory_query is also called  in this method.
It seems that we just need to free_top_overcommit_query here?